pkg/networkmanager/teamport.jsx

```diff
@@ -91,7 +91,7 @@ export const TeamPortDialog = ({ connection, dev, settings }) => {
                 : null}
             {teamMode == 'lacp'
                 ? <FormGroup fieldId={idPrefix + "-" + teamMode + "-key-input"} label={_("LACP key")}>
-                    <TextInput id={idPrefix + "-" + teamMode + "-key-input"} value={key} onChange={setKey} />
+                    <TextInput id={idPrefix + "-" + teamMode + "-key-input"} value={key} onChange={(_event, value) => setKey(value)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-161212-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)